### DISCUSSION TITLE
vue+qiankun使用loadMicroApp注册后，从模式为hash的子应用切回history模式的主应用hash被改

### DISCUSSION THREAD

**Discussion by shengshihua** [umijs/qiankun#1931] [Q&A]
vue+qiankun，主应用为history模式，子应用为hash模式。registerMicroApps默认注册，主子应用切回正常，但使用loadMicroApp注册后，打开子应用界面后再切回主应用，主应用原本的hash值自动从‘’改为‘#/’，导致界面出错。

**Comment by elricio** [comment]
同问，我history模式下也有相同的问题
